### PR TITLE
Moderator panel improvement

### PR DESCRIPTION
**Pull request by RazSKg**
Add users, owners to Moderator panel. Also add Category Tabs, improve Restaurants cards in Moderator panel and more minor changes...

### CODE REVIEW

File: easyrest/models/user.py [hunk 1]
```diff
@@ -33,6 +33,7 @@ class User(Base):
     birth_date = Column(Date)
     password = Column(Text)
     role_id = Column(Integer, ForeignKey('user_roles.id'), default=1)
+    is_active = Column(Integer, default=0)
```
Comment (serge30) [inline easyrest/models/user.py]: Probably [Boolean](https://docs.sqlalchemy.org/en/latest/core/type_basics.html#sqlalchemy.types.Boolean) is better here.

File: easyrest/scripts/json_helpers.py [hunk 2]
```diff
@@ -28,16 +31,26 @@ def wrap(data=[], success=True, error=None, message=None,):
...
     this function collide keys from "keys" parameter with values from "data" parameter to form dictionary
     :param data: list with data
-    :param keys: list of keys for formed dictionary
+    :param keys: tuple of keys for formed dictionary
     :return: dictionary with data from "data" and keys from "keys"
     """
     result = {}
     if len(data) != len(keys):
         return result
     for i in range(len(data)):
```
Comment (serge30) [inline easyrest/scripts/json_helpers.py]:
```
for key, value in zip(keys, values):
    if normalize_datetime:
        value = date_time_normalize(value)
    result[key] = value
```
looks much clearer

File: easyrest/views/moderator_controller.py [hunk 3]
```diff
@@ -128,3 +130,148 @@ def disapprove_restaurant_controller(request):
...
+                user.is_active,
+                user_status,
+                user.name,
+                user.phone_number,
+                user.email,
+                user.birth_date,
+                [restaurant.name for restaurant in user.restaurants]
+            ]
+            data.append(form_dict(user_data, keys, True))
+        wrap_data = wrap(data)
```
Comment (serge30) [inline easyrest/views/moderator_controller.py:198]: You'd better introduce some function which takes a model record and list of keys, and forms a dict:
```
def form_dict(record, keys):
    record = {}
    for key in keys:
        result[key] = getattr(record, key)
        # conver dates
    return result
```

Comment (RazSKg) [reply easyrest/views/moderator_controller.py:198]: I've use such approach to make dict with certain field aliases, e.g. in DB we have certain field names. But on frontend I send fields in other names. Due to this I created keys and order values in specific way. Can I leave this? (I will use "zip" as you suggest)

Comment (serge30) [reply easyrest/views/moderator_controller.py:198]: Up to you. But is there is any sense to have different field names on backend and frontend sides?

Comment (RazSKg) [reply easyrest/views/moderator_controller.py:198]: Yes, there are no big sense on it, but in this dict I also put values, which taken not from db query: such as "user_status" in this case.

Comment (serge30) [reply easyrest/views/moderator_controller.py:198]: You may form dict based on record and then add custom fields which are not in record:
```
data = form_dict(user, ("id", "is_active", "name", "phone_number", "email", "birth_date"))
data["status"] = user_status
...
```

Comment (RazSKg) [reply easyrest/views/moderator_controller.py:198]: ok

**Comment by RazSKg** [comment]
@serge30 fixed

File: easyrest/views/moderator_controller.py (hunk 3 above)
Comment (serge30) [reply easyrest/views/moderator_controller.py:198]: good, let's guys review your PR then. Please do not wait for my approve.

File: easyrest/views/moderator_controller.py [hunk 4]
```diff
@@ -128,3 +122,134 @@ def disapprove_restaurant_controller(request):
...
+        Json string(not pretty) created from dictionary with format:
+            {
+                "data": Null,
+                "success": success,
+                "error": error
+            }
+        If user is unauthorized and not an admin - throw 403:
+    """
+    data = request.json_body
+    id = data["id"]
```
Comment (MaxSukhovetskyi) [inline easyrest/views/moderator_controller.py]: Psl do not use id, use something like data_id

File: easyrest/views/moderator_controller.py [hunk 5]
```diff
@@ -128,3 +122,134 @@ def disapprove_restaurant_controller(request):
...
```
Comment (MaxSukhovetskyi) [inline easyrest/views/moderator_controller.py]: Same as above

File: frontend/src/components/ModeratorPanelComponents/Users.js [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+                                                    {restaurantName}
+                                                  </Typography>);})}
+                                              </TableCell>: null}
+                  <TableCell align="right">
+                    <Typography component="p" color={userInfo.is_active ? "primary" : "error"}>
+                        {userInfo.is_active ? "Active" : "Banned"}
+                    </Typography>  
+                  </TableCell>
+                  <TableCell align="right">
+                    {buttons(userInfo.id)[""+ userInfo.is_active]}
```
Comment (MaxSukhovetskyi) [inline frontend/src/components/ModeratorPanelComponents/Users.js]: Pls do userInfo.is_active.toString() for better understanding.

**Review by MaxSukhovetskyi — APPROVED** [review]

**Pull request merged**